dockerfiles/alpine/analytics/dashboard/Dockerfile

```diff
@@ -16,14 +16,82 @@
 #
 # ------------------------------------------------------------------------
 
-# set to product base image
-FROM wso2ei-analytics-base:6.4.0-alpine
+# set base Docker image to AdoptOpenJDK Alpine Docker image
+FROM adoptopenjdk/openjdk8:jdk8u212-b03-alpine
+LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
+      com.wso2.docker.source="https://github.com/wso2/docker-ei/releases/tag/v6.4.0.2"
```

Comment:
@Vathsan the upcoming Docker source release version for EI version `6.4.0` will be `v6.4.0.3`. Hence, let's update to `https://github.com/wso2/docker-ei/releases/tag/v6.4.0.3`. Please update the other files as well.

Reply:
Done in 532f659